configure.py

```diff
@@ -436,6 +436,10 @@ def process_command_line(args): # pylint: disable=too-many-locals,too-many-state
                            help='distribution specific version',
                            default='unspecified')
 
+    build_group.add_option('--linux-certstore-default-file', metavar='STRING',
+                           help='distribution specific path to pem file with trusted CAs',
+                           default='/etc/ssl/certs/ca-certificates.crt')
```

Comment:
So I decided I didn't like some parts of this logic, and it was faster to write the patch than it would be to explain, so https://github.com/randombit/botan/pull/1888. You can rebase onto that branch or just wait for it to get merged to master.